### ISSUE TITLE
Incorrect axis tick for VictoryStack

### ISSUE THREAD

**Issue by CherishIt** [FormidableLabs/victory#869]
I want to draw a stacked bar chart.
```
<VictoryChart>
  <VictoryStack>
    <VictoryBar
      data={[{x: "a", y: 2}, {x: "b", y: 3}, {x: "c", y: 5}]}
    />
    <VictoryBar
      data={[{x: "a", y: 1}, {x: "b", y: 4}, {x: "c", y: 5}]}
    />
    <VictoryBar
      data={[{x: "a", y: 3}, {x: "b", y: 2}, {x: "c", y: 6}]}
    />
  </VictoryStack>
</VictoryChart>
```
With above code, the output is 
![screen shot 2017-12-12 at 3 46 00 pm](https://user-images.githubusercontent.com/14925238/33872878-55613036-dede-11e7-9259-21413ae0df72.png)

But what I expected is the X axis ticks shows 'a', 'b', 'c', not '1.0', '1.5', ...

Are there anything I did wrong or somewhere I can config?

**Comment by buddhamangler** [comment]
I know this is old, but I am doing this very thing right now so I thought I would share.  I found that you have to specify your axes explicitly to get the tick values correct. Once you specify an axis you have to also give it the dependent axis so that will render the 'y' axis. You can generate your tickFormat list from your incoming data so it's not hard coded. Also if you want your first and last bar to have separation from the axis line and end of chart respectively you can use domainPadding={{x: [20, 20]}}.

`<VictoryStack>
...
   <VictoryAxis dependentAxis />
   <VictoryAxis tickFormat={{"a", "b", "c"}} />
...
`

**Comment by boygirl** [comment]
The issue with axis label formatting is fixed in the latest version (`victory@0.27.1`). The need for domainPadding is expected behavior. I'm going to close this issue, but please let me know if the problem persists.
